lib/shopify_app/configuration.rb

```diff
@@ -24,6 +25,18 @@ def initialize
       @myshopify_domain = 'myshopify.com'
     end
 
+    def session_repository=(klass)
+      if Rails.configuration.cache_classes
+        ShopifyApp::SessionRepository.storage = klass
+      else
+        reloader = defined?(ActiveSupport::Reloader) ? ActiveSupport::Reloader : ActionDispatch::Reloader
```

Comment:
Can we get rid of this now? Gem relies on >= 5.0 so we can assume `ActiveSupport::Reloader`